README.md

```diff
@@ -1,103 +1,68 @@
-<h1 align="center">
```

Comment:
aw, no more fancy README headers?

Reply:
We *can* do them, but my preference would be to use simple Markdown that can be read from the CLI without any HTML headaches. I'd love a Patchwork website though!

Reply:
yeah, that's fair, i personally like fancy README headers but it's only my subjective opinion, i know the trade-off is less interoperability using html instead of plain markdown. i reckon you should do what you prefer, and ignore me. :smiley_cat: